CMakeLists.txt

```diff
@@ -330,8 +330,10 @@ target_link_libraries(${PROJECT_NAME} ${LINK_LIBRARY_LIST})
 set_target_properties(${PROJECT_NAME} PROPERTIES POSITION_INDEPENDENT_CODE ON)
 set_target_properties(${PROJECT_NAME} PROPERTIES LINKER_LANGUAGE CXX)
 if(RPP_AUDIO_SUPPORT)
-    target_link_libraries(${PROJECT_NAME} ${PROJECT_SOURCE_DIR}/third_party/ffts/lib/libffts.a)
+    find_package(FFTS REQUIRED)
```

Comment:
Lets not use 'REQUIRED' else it will completely fail if not found. Instead do something like this with 'QUIET'.

It wil quietly find package, and if available, it will move ahead.
Else it will give a message that FFTS is required to build RPP with audio support. You can make it proceed with building RPP without audio support after that - like what happens at L338